server/room-battle.js

```diff
@@ -922,9 +940,9 @@ class RoomBattle {
 		this.playerCount++;
 		this.room.auth[user.userid] = Users.PLAYER_SYMBOL;
 		if (user.inRooms.has(this.id)) this.onConnect(user);
-		if (this.playerCount >= 2) {
-			// @ts-ignore
-			this.room.title = `${this.p1.name} vs. ${this.p2.name}`;
+		if (this.playerCount >= this.playerCap) {
+				// @ts-ignore
```

Comment:
```suggestion
			// @ts-ignore
```

I'm guessing this was a typo

Reply:
Nope, that's VS Code glitching out and not treating indentation changes as proper changes when I'm trying to only stage certain changes in a file. I know I said I'd leave this alone, but I think I'm just going to push what I have now since that will fix this anyway.